main.go

```diff
@@ -1,82 +1,62 @@
 package main
 
 import (
-	"github.com/hashicorp/go-getter"
-	"gopkg.in/go-playground/webhooks.v5/github"
-	"io"
-	"log"
+	"errors"
 	"net/http"
 	"os"
+	"strings"
 )
 
-const (
-	path = "/payload"
-	port = ":3069"
-)
-
-var (
-	Info    *log.Logger
-	Warning *log.Logger
-	Error   *log.Logger
-)
-
-func setLogStreams(
-	infoHandle io.Writer,
-	warningHandle io.Writer,
-	errorHandle io.Writer) {
-
-	Info = log.New(infoHandle,
-		"INFO: ",
-		log.Ldate|log.Ltime|log.Lshortfile)
-	Warning = log.New(infoHandle,
-		"WARNING: ",
-		log.Ldate|log.Ltime|log.Lshortfile)
-	Error = log.New(infoHandle,
-		"ERROR: ",
-		log.Ldate|log.Ltime|log.Lshortfile)
+func init() {
+	// Setup log streams
+	setLogStreams(os.Stdout, os.Stdout, os.Stderr)
 
 }
 
 func main() {
-	//Init Log streams
-	setLogStreams(os.Stdout, os.Stdout, os.Stderr)
+	//Initialize default ports
+	CIPort := "0.0.0.0:81"
+	websitePort := "0.0.0.0:80"
+
+	// Check for development flag
+	if len(os.Args) > 1 && os.Args[1] == "--development" {
+		CIPort = ":3081"
+		websitePort = ":3080"
+	}
```

Comment:
try if `os.Args` contains ...

Reply:
there is no contains method but I guess I could just write one.

Reply:
If I do `./deployment_manager random-junk --development`, then it shouldn't just silently ignore the development flag

Reply:
that makes sense, I'll add a contains check